website/pages/blog/announcing-cloudquery-snowflake-destination.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Announcing the CloudQuery Snowflake Destination Plugin
+tag: tutorial
+date: 2022/11/24
+description: >-
+  This tutorial will show you how to sync your cloud resources to a Snowflake database.
+author: yevgenypats
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+## Introduction
+
+Cloud Infrastructure data exploded over the last decade which created the need to store those in data-warehouses and datalakes for analysis and reporting enabled by cheap storage. Today I'm excited to announce the release of our new [Snowflake destination plugin](../docs/recipes/destinations/snowflake), which enables you to sync all supported CloudQuery [source plugins](../docs/plugins/sources) to a [Snowflake](https://www.sqlite.org/index.html) database.
```

Comment:
```suggestion
Cloud infrastructure data exploded over the last decade. Some teams using CloudQuery now collect data on more than 50 million (!) of their organization's cloud resources on a daily basis. This explosion has created the need to store this data in data warehouses and data lakes for better scaleability, analysis and reporting. Today I'm excited to announce the release of our new [Snowflake destination plugin](../docs/recipes/destinations/snowflake), which enables you to achieve this by syncing all supported CloudQuery [source plugins](../docs/plugins/sources) directly to a [Snowflake](https://www.sqlite.org/index.html) database.
```

Reply:
Thanks! Comited locally and pushed due to conflicts.